packages/twenty-server/src/workspace/workspace-sync-metadata/standard-objects/attachment.object-metadata.ts

```diff
@@ -80,4 +80,14 @@ export class AttachmentObjectMetadata extends BaseObjectMetadata {
   })
   @IsNullable()
   company: CompanyObjectMetadata;
+
+  @FieldMetadata({
+    type: FieldMetadataType.RELATION,
+    label: 'Opportunity',
+    description: 'Attachment opportunity',
+    icon: 'IconBuildingSkyscraper',
+    joinColumn: 'opportunityId',
+  })
+  @IsNullable()
+  opportunity: CompanyObjectMetadata;
```

Comment:
OpportunityObjectMetadata?

Reply:
Fortunately this has no impact today but yes good catch! 😄